source/configuration/program-detail-page.md

```diff
@@ -173,13 +173,13 @@ const leftPanel = {
 4. Dynamic links can be generated by passing a valid table filed to '{}'. For example, 'https://pubmed.ncbi.nlm.nih.gov/{pubmed_id}' shall link to 'https://pubmed.ncbi.nlm.nih.gov/29860917/'
 
 ## Configuring the Right Hand Panel
-In the current version of Bento, you can (a) add a donut widget that displays the counts by value of a key data entity in your data sharing platform and (b) add a count of files stored for the given program.
+In the current version of Bento donut widgets can be added to display the counts by value of any key data entity in the data sharing platform and (b) adding a count of files stored for a given program.
```

Comment:
I think we are missing a comma after Bento here.

Reply:
Accidentally remove. Added back in [539c65c](https://github.com/CBIIT/bento-docs/pull/103/commits/539c65c446c663426258e713986710db241dd70d).